website/widgets/form/table.md

```diff
@@ -352,3 +352,24 @@ q.page['example'] = ui.form_card(box='1 1 3 6', items=[
 ```
 
 Check the [complete example](/docs/examples/table-pagination) with all the supported features like sort/filter/search/download/reset.
+
+## With links
+
+Links can be displayed by setting `cell_type` to `'ui.url_table_cell_type'`. If `'new_tab'` is set to `'True'` the link will be opened on a new tab.
+To provide a label use the Markdown notation `[label](url)`.
+If the URL doesn't have '`http(s)://`' the link will be appended to the current URL, trying to access a page in the same domain.
+
+```py
+q.page['example'] = ui.form_card(box='1 1 3 3', items=[
+    ui.table(
+        name='table',
+        columns=[
+            ui.table_column(name='link', label='Link', cell_type=ui.url_table_cell_type(name='urls', new_tab=True))
```

Comment:
rename the attr.

Tip: Good check is to run `tools/showcase - make generate`. It's going to throw an error for these cases since it's an invalid code. However, make sure your UI build is always up to date prior to running it.